frontend/projects/upgrade/src/environments/environment.prod.ts

```diff
@@ -1,11 +1,11 @@
 export const environment = {
```

Comment:
we currently use this file for the all the beanstalk env builds... the `apiBaseUrl` and `googleClientId` will be safely swapped out, but `baseHrefPrefix` will cause an issue because it won't be replaced when running `build:prod`, so all the app routes will append '%BASE_HREF_PREFIX%'.

maybe a separate environment `environment.beanstalk.prod.ts` and configuration called `beanstalk` to keep these separate while we switch over?  then we can retain the current build process without the replacements, and point `build:prod` at that environment? `"build:prod": "ng build --configuration beanstalk",

Reply:
having a seperate config for beanstalk works. WIll update this PR with it.